mirror.rb

```diff
@@ -0,0 +1,10 @@
+def mirror(a)
+  og_length = a.length
+  (0..og_length-1).each do |i|
+    a << a[og_length - (i+ 1)]
+  end
+  return a
+end
```

Comment:
This was supposed to create a new array, not modify the original array - otherwise, looks good.